crates/cli/src/supporting/schema/settings.schema.json

```
@@ -348,6 +348,13 @@
 							"type": "string",
 							"enum": ["basic", "web", "oauth"],
 							"description": "The login method to use. One of: 'basic', 'web', or 'oauth'."
+						},
+						"localStorageKeys": {
+							"type": "array",
+							"description": "An array of localStorage keys to extract after login.",
```

this should also be moved to what is currently line 406 with this addition (in the "anyOf" for the object with the "web" option active), since it's only applicable for the web method.
```suggestion
							"description": "An array of localStorage keys to extract from the web view.",
```